rabbitmq/docker-command-smile-traefik.sh

```diff
@@ -38,19 +40,21 @@ export RABBITMQ_URL=amqp://${SERVER}
 export REDIS_URL=redis://redis
 
 # email notification
-#export EMAIL_HOST=<<email host>>
-#export EMAIL_PORT=465
-#export EMAIL_FROM_ADDRESS=<<email from address>>
-#export EMAIL_PASSWORD=<<email password>>
+export EMAIL_HOST=<<email host>>
+export EMAIL_PORT=465
+export EMAIL_FROM_ADDRESS=<<email from address>>
+export EMAIL_PASSWORD=<<email password>>
```

Comment:
This part is not required, maybe add a note; or keep them commented out

Reply:
commented out with the comment